GVFS/GVFS.NativeHooks.Common/common.posix.cpp

```diff
@@ -116,17 +116,19 @@ PIPE_HANDLE CreatePipeToGVFS(const PATH_STRING& pipeName)
     memset(&socket_address, 0, sizeof(struct sockaddr_un));
     
     socket_address.sun_family = AF_UNIX;
-    size_t resultLength = strlcpy(socket_address.sun_path, pipeName.c_str(), sizeof(socket_address.sun_path));
-    
-    if (resultLength >= sizeof(socket_address.sun_path))
+    size_t pathLength = strlen(pipeName.c_str());
```

Comment:
Same comment on `strnlen`, since it seems to be bounded by `sizeof(socket_address.sun_path)`

Reply:
We can do the same here, `std::string` already knows its length.

Reply:
And I notice on a second look that I wasn't leaving enough space for the trailing null, either.